Makefile

```
@@ -66,7 +66,7 @@ endif
 # Be aware that the target commands are only tested with Docker which is
 # scaffolded by default. However, you might want to replace it to use other
 # tools. (i.e. podman)
-CONTAINER_TOOL ?= docker
+CONTAINER_TOOL ?= podman
```

```suggestion
CONTAINER_TOOL="$(shell which podman >/dev/null 2>&1 && echo podman || echo docker)"
```
will add some flexibility in choosing `CONTAINER_TOOL`